test/e2e/config/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+bases:
```

Comment:
Added Kustomization but it doesn't seem to quite work yet.  Will look into this tomorrow.

Reply:
I added it but I don't think its ideal.  

I am new to Kustomize but I found that I could not apply patches that weren't in the same directory or below.  I originally tried to apply the patches that our present in config/default but that was not allowed.  The only thing I found that worked was to copy the existing patches and apply my imagePullPolicy.  I realize this isn't ideal due to duplication.  

I'm open for suggestions.

Reply:
envtest is used in integration tests, do we need it here? You can use
```
	cfg, _ := config.GetConfig()
	return client.New(cfg, client.Options{Scheme: scheme.Scheme})
```
to create a client.

What's more, I think you should wait for the webhook service ready before running e2e tests.

Reply:
@kerthcet pointed me in the right direction!